scripts/build/Jenkins/Jenkinsfile

```diff
@@ -750,6 +750,14 @@ finally {
             } else {
                 buildFailure = tm('${BUILD_FAILURE_ANALYZER}')
                 emailBody = "${BUILD_URL}\n${buildFailure}!"
+                if(env.SNS_TOPIC_GITHUB_ISSUE) {
```

Comment:
What do you think of changing this to SNS_TOPIC_BUILD_FAILURE for the topic name. This message can be used to trigger other tasks where we need the build failure results.

Reply:
Good point! I will change the topic name.